caproto/threading/client.py

```diff
@@ -546,15 +550,20 @@ def connected(self):
         return self.chid is not None and self.chid.connected
 
     def force_connect(self, pvname=None, chid=None, conn=True, **kws):
+        # not quite sure what this is for in pyepics, probably should
+        # be an arias for reconnect?
```

Comment:
Maybe it's my latent musical education, but I'm not sure "arias" means other than singing while waiting for the PV to connect.